src/main/java/org/cactoos/text/ReplacedText.java

```diff
@@ -47,13 +50,28 @@
     /**
      * The regular expression identifying the text to replace.
      */
-    private final Pattern regex;
+    private final Scalar<Pattern> regex;
 
     /**
      * The new replacement text.
      */
     private final Func<Matcher, String> replacement;
 
+    /**
+     * Shorthand ctor.
+     * <p>
+     * Will replace all instances of the substring matched by {@code find}
+     * with {@code replace}.
+     * @param text The text
+     * @param find The regular expression
+     * @param replace The replacement string
+     * @see #ReplacedText(Text, Pattern, Func)
```

Comment:
@llorllale Remove this line. We usually use `@see` for reference to other classes.